src/SymfonyCorp/Connect/Api/Api.php

```diff
@@ -32,15 +34,22 @@ class Api
 {
     const ENDPOINT = 'https://connect.symfony.com/api';
 
-    private $browser;
+    private $httpClient;
     private $parser;
     private $logger;
     private $endpoint;
     private $accessToken;
 
-    public function __construct($endpoint = null, Browser $browser = null, ParserInterface $parser = null, LoggerInterface $logger = null)
+    public function __construct($endpoint = null, HttpClientInterface $httpClient = null, ParserInterface $parser = null, LoggerInterface $logger = null)
```

Comment:
the type should be removed, otherwise this is a BC break as Browser cannot be passed anymore (and the next check is useless)